app/models/boat.rb

```diff
@@ -2,6 +2,7 @@ class Boat < ApplicationRecord
   # associations
   belongs_to :user
   has_many :bookings, dependent: :destroy
+  has_many :reviews
```

Comment:
has_many :reviews, dependent: :destroy

Reply:
If we delete a boat we should also delete it's reviews, I think

Reply:
i know i forgot that, we can add that later..